wordpress-mobile/WordPress-iOS
# Aztec: Wiring Post Settings

### Details:
This PR wires the Post Settings, in the new Aztec Editor's VC.

Closes #5936

### To test:
1. Launch the Aztec Editor
2. Tap over the top right navbar button (Ellipsis!)
3. Press over the **Options** action

Verify that the Post Settings screen shows up onscreen.

Needs review: @astralbodies 
Thanks in advance!
